parlai/core/metrics.py

```diff
@@ -23,6 +24,20 @@
     # We'll just turn off things, but we might want to warn the user
     nltkbleu = None
 
+try:
+    import rouge as rouge
+except ImportError:
+    # User doesn't have rouge installed, so we can't use it for rouge
+    # We'll just turn off things, but we might want to warn the user
+    warn_once('Run pip install py-rouge to get rouge metrcis')
+    rouge = None
+
+try:
+    import nltk.tokenizer  # noqa
```

Comment:
I don't know if that is acceptable in the code though ?  @stephenroller 
In order to disable the import unused lint F401

Reply:
ya make it `import nltk.tokenizer  # noqa: F401`

Reply:
But I don't think this is right:

```python-traceback
>>> nltk.tokenize.sent_tokenize("Test.")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/circleci/venv/lib/python3.6/site-packages/nltk/tokenize/__init__.py", line 104, in sent_tokenize
    tokenizer = load('tokenizers/punkt/{0}.pickle'.format(language))
  File "/home/circleci/venv/lib/python3.6/site-packages/nltk/data.py", line 870, in load
    opened_resource = _open(resource_url)
  File "/home/circleci/venv/lib/python3.6/site-packages/nltk/data.py", line 995, in _open
    return find(path_, path + ['']).open()
  File "/home/circleci/venv/lib/python3.6/site-packages/nltk/data.py", line 701, in find
    raise LookupError(resource_not_found)
LookupError:
**********************************************************************
  Resource punkt not found.
  Please use the NLTK Downloader to obtain the resource:

  >>> import nltk
  >>> nltk.download('punkt')

  For more information see: https://www.nltk.org/data.html

  Attempted to load tokenizers/punkt/PY3/english.pickle

  Searched in:
    - '/home/circleci/nltk_data'
    - '/home/circleci/venv/nltk_data'
    - '/home/circleci/venv/share/nltk_data'
    - '/home/circleci/venv/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
    - ''
**********************************************************************
```

So I think you need to be catching a LookupError, and you need to be doing it when you actually run rouge.